reference/platforms.md

```diff
@@ -10,9 +10,9 @@
 | DAM        | Digital Asset Management              |
 | MA         | Marketing Automation                  |
 | POS        | Point of Sale                         |
-| Search     | and Findability                       |
```

Comment:
Does it make sense for this to refer to Discoverability in addition to or in place of Findability? We've been an Algolia partner for years, and I've generally found Discoverability to be a common term in search and discovery conversations, with search and Findability being more synonymous. Discovery is also used by Gartner when discussing the category.

I always refer back to this old post from Seth Godin. 
https://seths.blog/2014/04/search-vs-discovery/

"Discovery, on the other hand, is what happens when the universe (or an organization, or a friend) helps you encounter something you didn't even know you were looking for."

Modern search platforms aid not only in Search (Findability) but also in Discoverability.

Reply:
Good point - updated, thanks!